### PR TITLE
#244 / Reusable Key Figure Card Component

### PR DESCRIPTION
**Pull request by AshleyN2**
# Description

* Worked on this with @Calvin-Ngugi 

This section contains a type of card that is used to show various metrics.
The top part can show either Key Figure or logo Image.

The card shows the following inputs:
```
1. Figure (string)
2. Description (string) which is optional
3. isImage (boolean)
```

To achieve this, we needed to:

```
- Export the component for re-usability
- Make it visible on the page so users can notice it easily
```

Fixes # (issue) - *Example*: Fixes # ([S4Y-400](https://www.w3.org/Provider/Style/dummy.html), [S4Y-100](https://www.w3.org/Provider/Style/dummy.html))

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot 
Screenshot from 2023-02-27 10-14-52

Responsiveness

Screenshot from 2023-02-27 10-15-17

# How Has This Been Tested?
Tested by rendering different inputs. 

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AshleyN2** [comment]
@IanOdhiambo9 kindly review.

File: libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- <p>elewa-key-figure-card works!</p> -->
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.html:1]: Remove code that is not being used.

File: libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+<div class="card">
+    <div class="details">
+        <div class="img" isImage >
+            <img class="fig">{{figure}}<img>
+            <p class="text">{{description}}</p>
+        </div>
+    </div>
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.html:8]: use structural directive

File: libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+@Component({
+  selector: 'elewa-group-elewa-key-figure-card',
+  templateUrl: './elewa-key-figure-card.component.html',
+  styleUrls: ['./elewa-key-figure-card.component.css']
+})
+export class ElewaKeyFigureCardComponent {
+  @Input() figure? : string;
+  @Input() description? : string;
+  @Input() isImage? : boolean;
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.ts:11]: only description could be optional

**Review by IanOdhiambo9 — COMMENTED** [review]
this issue has been recently completed

**Comment by IanOdhiambo9** [comment]
This issue has been merged

**Pull request closed without merging**